WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormTableViewDataSource.swift

```diff
@@ -277,13 +279,31 @@ private extension ProductFormTableViewDataSource {
         cell.hideSeparator()
     }
 
+    func configurePromoteWithBlaze(cell: UITableViewCell) {
+        guard let cell = cell as? LeftImageTableViewCell else {
+            fatalError()
+        }
+        let title = NSLocalizedString(
+            "Promote with Blaze",
+            comment: "Label for button on product form to open Blaze flow"
+        )
+
+        cell.configure(
+            image: .blaze,
+            imageSize: CGSize(width: 20, height: 20),
+            text: title,
+            textColor: .accent
+        )
+        cell.hideSeparator()
+    }
+
     func configureSeparator(cell: UITableViewCell) {
         guard let cell = cell as? SpacerTableViewCell else {
             fatalError("Unexpected table view cell for the separator cell")
         }
         cell.selectionStyle = .none
         cell.backgroundColor = .listBackground
-        cell.showSeparator()
+        cell.hideSeparator()
```

Comment:
If I look at the current design for Blaze Optimization, there is no line above/below separator, so I changed it to follow it. Doing so makes the Blaze button looks correct, too.

<img width="821" alt="Screenshot 2023-10-23 at 20 29 32" src="https://github.com/woocommerce/woocommerce-ios/assets/266376/1bf2dc52-9bf2-4cc6-8d06-a5eeff53da01">